src/pages/LogDrop/FileDropper.tsx

```diff
@@ -42,7 +42,14 @@ const FileDropper: React.FC = () => {
   const onParse = useCallback(
     (logType: LogTypes | undefined) => {
       if (logType) {
-        setLogMetadata({ isUploadedLog: true, logType });
+        setLogMetadata({
+          isUploadedLog: true,
+          logType,
+          renderingType:
+            logType === LogTypes.RESMOKE_LOGS
+              ? SupportedLogRenderingTypes.Resmoke
+              : SupportedLogRenderingTypes.Default,
```

Comment:
Doing the above we can also avoid this.

Reply:
hmm i think this may be ok, Since this could be used to distinguish between log types for log sectioning project? I do think it would be important to iron that out in your project or at some point before it.